# Dynamixel setup
motor_ids: [1, 2]
port: "/dev/ttyUSB0"
baudrate: 57600
control_mode: "position"

# Where calibration is saved
calibration_path: "config/gripper_calibration.yaml"

# Motion range limits (per motor)
min_position_rad:
  1: 0.0
  2: 0.0
max_position_rad:
  1: 3.14
  2: 3.14

# Geometric tendon setup (for pull-length to rotation)
pulley_radius:
  1: 20.0
  2: 20.0

# Joint Radius (for pull-length to rotation)
joint_radius:
  1: 5.0
  2: 5.0

# Link lengths for IK or forward modeling
link1_length: 60
link2_length: 60

# Optional: Max joint angles (used in IK or joint clamping)
max_joint_angles_rad:
  1: 1.57
  2: 1.57

# Here we put the different gripper configurations
open_positions:
  1: 3.1
  2: 3.1

closed_positions:
  1: 1.0
  2: 1.0